Dockerfile

```diff
@@ -47,17 +47,16 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 # Install python 2.7 and dependancies for Genetorrent.
 #
 
-RUN apt-get install -y python && \
-    wget -qO- https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz#md5=f50e08ef0fe55178479d3a618efe21db | \
-    tar xvz --strip-components 1 && \
-    python setup.py install && \
-    mkdir -p /icgc/cli/clients
+RUN apt-get install -y python-pip && \
+    pip install -U pip setuptools
 
-COPY /lib/cli/ /icgc/cli/
+COPY . /icgc/icgcget/
+COPY /conf/ /icgc/conf
```

Comment:
At this point there is no configuration in /icgc/icgcget/conf.  I may be moving the files there when implementing layered config, but I'll change the dockerfile when that happens.

Also, I apologize for taking so long to respond:  I'm not reciving the email alerts for some reason.